tree_traversal_without recursion.py

```diff
@@ -0,0 +1,63 @@
+class stack:
+    def __init__(self):
+        self.stack = []
+    def push(self, item):
+        self.stack.append(item)
+    def peek(self):
+        if self.isEmpty():
+            return None
+        else:
+            return self.stack[-1]
+    def pop(self):
+        if self.isEmpty():
+            return None
+        else:
+            return self.stack.pop()
+    def size(self):
+        return len(self.stack)
+    def isEmpty(self):
+        return self.size() == 0
+
+class Node:
+    def __init__(self, data):

```

Comment:
You can make this API a lot better by using default parameters:
def __init__(self, data, right = None, left = None):